doc/developers/contributing.rst

```diff
@@ -698,15 +698,15 @@ Here's a simple example of code using some of the above guidelines::
 
         Parameters
         ----------
-        X : array-like, shape = (n_samples, n_features)
+        X : array-like, shape = [n_samples, n_features]
```

Comment:
I would write:

`X : array-like, shape (n_samples, n_features)`

without =

and below

`x : numpy array, shape (n_features,)`

the reason is that shape is a tuple not a list.

Regarding the = I don't think it's necessary. We also tend not to put it:

```
alex@:scikit-learn(master)$ git grep "shape (n_sam" | wc
     894    6823   79882
alex@:scikit-learn(master)$ git grep "shape = (n_sam" | wc
      97     848    8815
```

Reply:
I was also wondering if it should be `array_like` vs. `array-like`?

Numpy seems to be using `array_like` and it looks better to me as it's a correct variable name.

Reply:
+1 for "array-like". There is not reason for it to be a correct Python variable name.